### PR TITLE
Frontend web development html css js

### PR DESCRIPTION
**Pull request by darshan4903**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #issue_number_here 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [ ] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Add only one md file.

**Comment by darshan4903** [comment]
deleted @kashishahuja2002

File: Web_Development/FrontEnd/HTML detail tag/HTML detailtag.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# HTML detail tag
+It can be used when we need create a widget in which initially content is hidden and it displays the content when user clicks on it. The content of the details tag is visible when open the set attributes. The summary tag is used with the details tag for specifying visible heading. 
+
+# Attributes
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML detail tag/HTML detailtag.md:4]: Wrong level of heading, use ##.

File: Web_Development/FrontEnd/HTML detail tag/HTML detailtag.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+`open` : This is basically a boolean attribute which tells whether it will show details or not if attribute exists then the elements of detail will be visible and if attribute is not present then it will hide the elements of detail tag.
+### Syntax
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML detail tag/HTML detailtag.md:6]: Wrong level of heading, use ##.

File: Web_Development/FrontEnd/HTML detail tag/HTML detailtag.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+```html
+<details>
+    <summary>content</summary>
+    <div> hidden content </div>
+</details>
+```
+
+### Example for detail tag
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML detail tag/HTML detailtag.md:14]: Wrong level of heading, use ##.

File: Web_Development/FrontEnd/HTML detail tag/HTML detailtag.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+#### Initial look (when content is hidden)
+<img width="1439" alt="Screenshot 2021-11-14 at 8 33 42 PM" src="https://user-images.githubusercontent.com/84950175/141691655-c96ba1f6-c11a-46ae-973d-b2135b80b0cd.png">
+
+#### Final look (when content is displayed when user clicked on it)
+<img width="1439" alt="Screenshot 2021-11-14 at 8 33 54 PM" src="https://user-images.githubusercontent.com/84950175/141691763-6c90f040-26ff-4869-8e5e-e6a09d06035b.png">
+
+#### Code for above Example
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML detail tag/HTML detailtag.md:21]: Don't make it a heading, just bold.
Change text to: "Code:"

File: Web_Development/FrontEnd/HTML detail tag/HTML detailtag.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+</details>
+```
+
+### Example for detail tag
+#### Initial look (when content is hidden)
+<img width="1439" alt="Screenshot 2021-11-14 at 8 33 42 PM" src="https://user-images.githubusercontent.com/84950175/141691655-c96ba1f6-c11a-46ae-973d-b2135b80b0cd.png">
+
+#### Final look (when content is displayed when user clicked on it)
+<img width="1439" alt="Screenshot 2021-11-14 at 8 33 54 PM" src="https://user-images.githubusercontent.com/84950175/141691763-6c90f040-26ff-4869-8e5e-e6a09d06035b.png">
+
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML detail tag/HTML detailtag.md:20]: Instead of these screenshots, add a video output.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]

**Comment by darshan4903** [comment]
Updated @kashishahuja2002

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
LGTM. Tag the supervisor of your batch to further review and merge the PR.

**Review by kashishahuja2002 — APPROVED** [review]

**Comment by darshan4903** [comment]
@2024-SANDHYA please merge this PR as it os approved

**Comment by 2024-SANDHYA** [comment]
> 
> 
> @2024-SANDHYA please merge this PR as it os approved

I am not a supervisor though, but still, lemme check.

**Pull request merged**